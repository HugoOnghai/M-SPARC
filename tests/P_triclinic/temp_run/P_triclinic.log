
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat P_triclinic.inpt
# nprocs: 24

# Test: P_triclinic
LATVEC:
0.970404375040354    0.000000000000000  -0.241485711590850
-0.504751331827941   0.862706058268719  -0.031053985965896
0.000000000000000    0.000000000000000   1.000000000000000
CELL: 7.5 8.1 8.2
MESH_SPACING: 0.20
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.775131
TOL_SCF: 1e-6
CALC_PRES: 1
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none
<\INPT>

 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 4
 Default atomic mass for P is 30.973762
 pseudo_dir # 1 = P.psp8

 COORD:
   4.533622500000000   6.074999999999999   2.334212000000000
   2.966377500000000   2.025000000000000   5.865787999999999
   7.075117500000000   6.074999999999999   6.789804999999999
   0.424882500000000   2.025000000000000   1.410195000000000

   0.006063285313173

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 17
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (12.353 sec)

 Estimated memory usage:
 Total:  206.35 MB
 orbitals            :   32.33 MB
 sparse matrices     :  160.71 MB
 global-size vectors :    6.66 MB
 mixing histories    :    6.66 MB

 Finding rb for P ...
 rb = {2.961 2.797 2.797}, int_b = -4.999640288583016, err_rb = 3.597e-04
 rb = {3.158 2.995 2.995}, int_b = -4.999788885929228, err_rb = 2.111e-04
 rb = {3.355 3.192 3.192}, int_b = -4.999936175871547, err_rb = 6.382e-05
 rb = {3.553 3.389 3.389}, int_b = -4.999992225924533, err_rb = 7.774e-06
 rb = {3.750 3.587 3.587}, int_b = -5.000014823728328, err_rb = 1.482e-05
 rb = {3.947 3.784 3.784}, int_b = -5.000021053321354, err_rb = 2.105e-05
 rb = {4.145 3.982 3.982}, int_b = -5.000021433992497, err_rb = 2.143e-05
 rb = {4.342 4.179 4.179}, int_b = -5.000019555784151, err_rb = 1.956e-05
 rb = {4.539 4.376 4.376}, int_b = -5.000017286634148, err_rb = 1.729e-05
 rb = {4.737 4.574 4.574}, int_b = -5.000015139670080, err_rb = 1.514e-05
 rb = {4.934 4.771 4.771}, int_b = -5.000012388270229, err_rb = 1.239e-05
 rb = {5.132 4.968 4.968}, int_b = -5.000011055970941, err_rb = 1.106e-05
 rb = {5.329 5.166 5.166}, int_b = -5.000010692056292, err_rb = 1.069e-05
 rb = {5.526 5.363 5.363}, int_b = -5.000010179931414, err_rb = 1.018e-05
 rb = {5.724 5.561 5.561}, int_b = -5.000009680179700, err_rb = 9.680e-06
 rb = {5.921 5.758 5.758}, int_b = -5.000009496384155, err_rb = 9.496e-06
 rb = {6.118 5.955 5.955}, int_b = -5.000009593401172, err_rb = 9.593e-06
 rb = {6.316 6.153 6.153}, int_b = -5.000010050436990, err_rb = 1.005e-05
 rb = {6.513 6.350 6.350}, int_b = -5.000007261081358, err_rb = 7.261e-06
 rb = {6.711 6.547 6.547}, int_b = -5.000005970582165, err_rb = 5.971e-06
 rb = {6.908 6.745 6.745}, int_b = -5.000006735633090, err_rb = 6.736e-06
 rb = {7.105 6.942 6.942}, int_b = -5.000001221026189, err_rb = 1.221e-06
 rb = {7.303 7.139 7.139}, int_b = -4.999999888461999, err_rb = 1.115e-07
 rb = {7.500 7.337 7.337}, int_b = -5.000000025021889, err_rb = 2.502e-08
 rb = {7.697 7.534 7.534}, int_b = -4.999999993625151, err_rb = 6.375e-09
 rb = {7.697 7.534 7.534}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 19.999999971491

 Integration b_ref = 20.000000000029

 ****************************************
 *          Eself_ref = 178.657146       *
 ****************************************
 Done. (5.290796 s)
 Time for b calculation: 5.304 seconds.

 Starting calculating nonlocal projectors ...
 Done. (2.495390 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 2.47677e-09 in 49 iterations.
 Poisson problem took 5.540169s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 5.380996
 This SCF iteration took 7.997 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.906324
 This SCF iteration took 4.815 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.457967
 This SCF iteration took 4.573 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.379559
 AAR converged to a relative residual of 4.87186e-09 in 43 iterations.
 Poisson problem took 4.623327s
 Escc = 0.54589015
 Etot = -26.13839988
 Eatom = -6.53459997
 Error in SCF iteration: 4.1063e-02 
 This SCF iteration took 9.336 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.372624
 AAR converged to a relative residual of 3.25944e-09 in 43 iterations.
 Poisson problem took 4.626050s
 Escc = 0.21385253
 Etot = -26.17916111
 Eatom = -6.54479028
 Error in SCF iteration: 4.9938e-02 
 This SCF iteration took 9.291 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.367856
 AAR converged to a relative residual of 8.74241e-09 in 37 iterations.
 Poisson problem took 3.979423s
 Escc = 0.12373126
 Etot = -26.23920375
 Eatom = -6.55980094
 Error in SCF iteration: 1.9441e-02 
 This SCF iteration took 8.751 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.366499
 AAR converged to a relative residual of 9.71378e-09 in 36 iterations.
 Poisson problem took 3.872352s
 Escc = 0.08002733
 Etot = -26.24508759
 Eatom = -6.56127190
 Error in SCF iteration: 1.3521e-02 
 This SCF iteration took 8.499 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.366777
 AAR converged to a relative residual of 5.12773e-09 in 37 iterations.
 Poisson problem took 3.944789s
 Escc = -0.00870539
 Etot = -26.24807889
 Eatom = -6.56201972
 Error in SCF iteration: 4.3009e-03 
 This SCF iteration took 8.618 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.366144
 AAR converged to a relative residual of 8.29015e-09 in 32 iterations.
 Poisson problem took 3.432984s
 Escc = 0.00462606
 Etot = -26.24832269
 Eatom = -6.56208067
 Error in SCF iteration: 2.3176e-03 
 This SCF iteration took 8.126 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.366139
 AAR converged to a relative residual of 5.04809e-09 in 31 iterations.
 Poisson problem took 3.308532s
 Escc = -0.00857800
 Etot = -26.24834888
 Eatom = -6.56208722
 Error in SCF iteration: 1.1875e-03 
 This SCF iteration took 8.012 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.365941
 AAR converged to a relative residual of 4.18613e-09 in 31 iterations.
 Poisson problem took 3.329573s
 Escc = -0.00022356
 Etot = -26.24835816
 Eatom = -6.56208954
 Error in SCF iteration: 8.5003e-04 
 This SCF iteration took 8.015 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.365958
 AAR converged to a relative residual of 9.34752e-09 in 28 iterations.
 Poisson problem took 2.976912s
 Escc = -0.00000185
 Etot = -26.24837048
 Eatom = -6.56209262
 Error in SCF iteration: 2.5156e-04 
 This SCF iteration took 7.625 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.365963
 AAR converged to a relative residual of 6.37841e-09 in 25 iterations.
 Poisson problem took 2.685674s
 Escc = -0.00014345
 Etot = -26.24837147
 Eatom = -6.56209287
 Error in SCF iteration: 8.1331e-05 
 This SCF iteration took 7.309 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.365960
 AAR converged to a relative residual of 2.01761e-09 in 25 iterations.
 Poisson problem took 2.654952s
 Escc = 0.00002131
 Etot = -26.24837150
 Eatom = -6.56209288
 Error in SCF iteration: 2.5085e-05 
 This SCF iteration took 7.264 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = 0.365961
 AAR converged to a relative residual of 2.5854e-09 in 19 iterations.
 Poisson problem took 2.026471s
 Escc = 0.00004545
 Etot = -26.24837146
 Eatom = -6.56209286
 Error in SCF iteration: 1.2777e-05 
 This SCF iteration took 6.606 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = 0.365961
 AAR converged to a relative residual of 9.96874e-09 in 14 iterations.
 Poisson problem took 1.485477s
 Escc = 0.00000539
 Etot = -26.24837150
 Eatom = -6.56209288
 Error in SCF iteration: 2.9687e-06 
 This SCF iteration took 6.164 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = 0.365961
 AAR converged to a relative residual of 3.07602e-09 in 13 iterations.
 Poisson problem took 1.388539s
 Escc = 0.00000249
 Etot = -26.24837140
 Eatom = -6.56209285
 Error in SCF iteration: 1.1796e-06 
 This SCF iteration took 6.020 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 15 
 ========================= 
 Fermi energy = 0.365962
 AAR converged to a relative residual of 8.4432e-09 in 8 iterations.
 Poisson problem took 0.849516s
 Escc = -0.00000012
 Etot = -26.24837150
 Eatom = -6.56209288
 Error in SCF iteration: 4.1746e-07 
 This SCF iteration took 5.514 s.


 Finished SCF iteration in 15 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -26.248371500 Ha.       *
 *          Energy per atom = -6.562092875 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 5.288 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
   0.049301900825631  -0.044741870067739  -0.008597676307864
  -0.049301994939488   0.044741424765857   0.008598027659083
  -0.041117705309088   0.039617112212530   0.001376147084617
   0.041117799422945  -0.039616666910647  -0.001376498435836

 Max magnitude of forces (Ha/Bohr):   0.067129966483483


 Time for calculating forces: 5.686862 s.

["Stress in GPa"


] -63.207612408308897   3.428779921565412  30.883219069235874
   3.428779921565412 -73.169466749181012  35.898501822552888
  30.883219069235874  35.898501822552888  14.816430202057155


 Time for calculating stress: 18.354068 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  4.533622 	  6.075000 	  2.334212 
  2.966378 	  2.025000 	  5.865788 
  7.075118 	  6.075000 	  6.789805 
  0.424882 	  2.025000 	  1.410195 

 Run-time of the program: 184.956462 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.837173733317784
                        L1: 7.500000000000000
                        L2: 8.100000000000000
                        L3: 8.199999999999999
                        Nx: 38
                        Ny: 40
                        Nz: 41
                         N: 62320
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [0 0 0]
                  kptshift: [0 0 0]
                      nkpt: [1 1 1]
                     tnkpt: 1
                      wkpt: 1
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 17
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157751310000000e+02
                       bet: 9.999996636976061e+02
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 20
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 3.996928357612732e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 1
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'P_triclinic'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x1 struct]
                     n_atm: 4
                     Atoms: [4x3 double]
           mvAtmConstraint: [4x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [4x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.197368421052632
                        dy: 0.202500000000000
                        dz: 0.200000000000000
                        dV: 0.006691882144613
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [62320x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [494x1 logical]
                 isOutr_11: [494x1 logical]
                      I_11: [494x1 double]
                     II_11: [494x1 double]
                      V_11: [494x1 double]
                 isOutl_22: [520x1 logical]
                 isOutr_22: [520x1 logical]
                      I_22: [520x1 double]
                     II_22: [520x1 double]
                      V_22: [520x1 double]
                 isOutl_33: [533x1 logical]
                 isOutr_33: [533x1 logical]
                      I_33: [533x1 double]
                     II_33: [533x1 double]
                      V_33: [533x1 double]
                  isOutl_1: [456x1 logical]
                  isOutr_1: [456x1 logical]
                       I_1: [456x1 double]
                      II_1: [456x1 double]
                       V_1: [456x1 double]
                  isOutl_2: [480x1 logical]
                  isOutr_2: [480x1 logical]
                       I_2: [480x1 double]
                      II_2: [480x1 double]
                       V_2: [480x1 double]
                  isOutl_3: [492x1 logical]
                  isOutr_3: [492x1 logical]
                       I_3: [492x1 double]
                      II_3: [492x1 double]
                       V_3: [492x1 double]
                 G_JOutl_1: [747840x1 logical]
                 G_JOutr_1: [747840x1 logical]
                     G_J_1: [747840x1 double]
                     G_I_1: [747840x1 double]
                     G_V_1: [747840x1 double]
                 G_JOutl_2: [747840x1 logical]
                 G_JOutr_2: [747840x1 logical]
                     G_J_2: [747840x1 double]
                     G_I_2: [747840x1 double]
                     G_V_2: [747840x1 double]
                 G_JOutl_3: [747840x1 logical]
                 G_JOutr_3: [747840x1 logical]
                     G_J_3: [747840x1 double]
                     G_I_3: [747840x1 double]
                     G_V_3: [747840x1 double]
                   Lap_std: [62320x62320 double]
                    grad_1: [62320x62320 double]
                    grad_2: [62320x62320 double]
                    grad_3: [62320x62320 double]
                LapPreconL: [62320x62320 double]
                LapPreconU: [62320x62320 double]
              memory_usage: 216375048
               staticfname: 'P_triclinic.static'
                 suffixNum: 0
                  outfname: 'P_triclinic.out'
                  parallel: 0
                         b: [62320x1 double]
                     b_ref: [62320x1 double]
                    rho_at: [62320x1 double]
                     Eself: 1.547915845208721e+02
                 Eself_ref: 1.786571463615903e+02
                       V_c: [62320x1 double]
                 PosCharge: 19.999999971491494
                 NegCharge: -19.999999971491494
                    E_corr: -1.394411065405166e-04
                       rho: [62320x1 double]
                      Atom: [1x4 struct]
                       phi: [62320x1 double]
                      e_xc: [62320x1 double]
                 dvxcdgrho: [62320x1 double]
                       Vxc: [62320x1 double]
                      Veff: [62320x1 double]
                         X: [62320x7 double]
                         F: [62320x7 double]
          mixing_hist_fkm1: [62320x1 double]
          mixing_hist_xkm1: [62320x1 double]
                       psi: [62320x17 double]
    upper_bound_guess_vecs: [62320x1 double]
                    EigVal: [17x1 double]
                  lambda_f: 0.365961598781435
                       occ: [17x1 double]
                    Etotal: -26.248371500134766
                     Eband: 3.013791031605544
                       Exc: -6.959440573599752
                    Exc_dc: -9.058941755679786
                  Eelec_dc: 1.234301318876221e+02
                      Eent: -7.151509574840790e-05
                      Escc: -1.243680353537774e-07
                   S_Debug: [1x1 struct]
                     force: [4x3 double]
                 abs_force: [4x1 double]
                    Stress: [3x3 double]
                  Pressure: 40.520216318477587
             stress_dimred: [3x3 double]

