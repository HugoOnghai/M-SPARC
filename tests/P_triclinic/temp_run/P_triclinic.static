Atomic forces (Ha/Bohr):
      0.0493019008      -0.0447418701      -0.0085976763
     -0.0493019949       0.0447414248       0.0085980277
     -0.0411177053       0.0396171122       0.0013761471
      0.0411177994      -0.0396166669      -0.0013764984
Stress (GPa):
 -6.3207612408E+01   3.4287799216E+00   3.0883219069E+01 
  3.4287799216E+00  -7.3169466749E+01   3.5898501823E+01 
  3.0883219069E+01   3.5898501823E+01   1.4816430202E+01 
