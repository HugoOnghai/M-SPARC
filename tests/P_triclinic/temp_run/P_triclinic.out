***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:51                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.500000 8.100000 8.200000 
LATVEC:
0.970404375040354 0.000000000000000 -0.241485711590850 
-0.504751331827941 0.862706058268719 -0.031053985965896 
0.000000000000000 0.000000000000000 1.000000000000000 
FD_GRID: 38 40 41
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 35
NSTATES: 17
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: P_triclinic.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.197368 (Bohr)
Mesh spacing in y-direction        :  0.202500 (Bohr)
Mesh spacing in z direction        :  0.200000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  P_triclinic.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  20
Atom type 1  (valence electrons)   :  P 5
Pseudopotential                    :  P.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.70 7.53 7.53
Number of atoms of type 1          :  4
Estimated total memory usage       :  206.35 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -6.5345999705E+00        4.106E-02        26.719
2            -6.5447902765E+00        4.994E-02        9.289
3            -6.5598009382E+00        1.944E-02        8.750
4            -6.5612718967E+00        1.352E-02        8.496
5            -6.5620197221E+00        4.301E-03        8.616
6            -6.5620806715E+00        2.318E-03        8.124
7            -6.5620872195E+00        1.187E-03        8.010
8            -6.5620895408E+00        8.500E-04        8.013
9            -6.5620926206E+00        2.516E-04        7.623
10           -6.5620928671E+00        8.133E-05        7.307
11           -6.5620928750E+00        2.508E-05        7.263
12           -6.5620928648E+00        1.278E-05        6.604
13           -6.5620928754E+00        2.969E-06        6.162
14           -6.5620928500E+00        1.180E-06        6.018
15           -6.5620928750E+00        4.175E-07        5.512
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -6.5620928750E+00 (Ha/atom)
Total free energy                  : -2.6248371500E+01 (Ha)
Band structure energy              :  3.0137910316E+00 (Ha)
Exchange correlation energy        : -6.9594405736E+00 (Ha)
Self and correction energy         : -1.5479172396E+02 (Ha)
Entropy*kb*T                       : -7.1515095748E-05 (Ha)
Fermi level                        :  3.6596159878E-01 (Ha)
Average force                      :  6.2122168543E-02 (Ha/Bohr)
Maximum force                      :  6.7129966483E-02 (Ha/Bohr)
Time for force calculation         :  5.687 (sec)
Pressure                           :  4.0520216318E+01 (GPa)
Max stress                         :  7.3169466749E+01 (GPa)
Time for stress calculation        :  18.356 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  184.956 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
